package.json

```diff
@@ -4,7 +4,8 @@
   "description": "OCSP Stapling implementation",
   "main": "lib/ocsp.js",
   "scripts": {
-    "test": "mocha test/*-test.js && jshint lib/*.js lib/**/*.js && jscs lib/*.js lib/**/*.js test/*.js test/**/*.js"
+    "lint": "jshint lib/*.js lib/**/*.js && jscs lib/*.js lib/**/*.js test/*.js test/**/*.js",
+    "test": "mocha test/*-test.js"
```

Comment:
That's already been added to the .travis.yml, so it **will** get run by Travis on every PR. This change permits devs to run 'npm test' locally without having to run the lint tests every time.